### PR TITLE
Added Issue Template

### PR DESCRIPTION
**Pull request by ferranpons**
Need because we don't receive so many info about problems submitted.

### CODE REVIEW

**Comment by rocboronat** [comment]
👍

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Which version of the library do you actually use?
+
+- Do you have the localization permission granted?
+
+- Are you sending parameters to the activity through the bundle?
+
+- Could you describe what are the actions do you make to raise that error?
+
+- Android monitor shows any log related to the library when the error is shown?
```
Comment (rocboronat) [inline .github/ISSUE_TEMPLATE.md]: We could clarify what's the TAG of Leku, to help people reaching the Leku's log messages

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## Description
+
+## Info Required
+
+- Which version of the library do you actually use?
+
+- Do you have the localization permission granted?
```
Comment (alorma) [inline .github/ISSUE_TEMPLATE.md]: Maybe i will use Checkboxes for that:

[ ] Do you have the localization permission granted?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ferranpons) [inline .github/ISSUE_TEMPLATE.md]: maybe, but the exceptions include the activity class name for the log. For the rest there are no logs or stack traces

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (ferranpons) [inline .github/ISSUE_TEMPLATE.md]: only for one thing?

**Comment by ferranpons** [comment]
@rocboronat @alorma @gerardpedreny If you have any suggestion or you feel that is information missing please don't hesitate to comment it or create a PR.

Many thanks to you guys!

**Comment by rocboronat** [comment]
You're welcome, @ferranpons ! :·)

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (rocboronat) [inline .github/ISSUE_TEMPLATE.md]: OK! I understood that this also apply to requests or something like that, not only crashes :D

**Pull request merged**